content/en/threat-model.md

```diff
@@ -0,0 +1,123 @@
+|                        | x                |               |                       |              |               |              | x           | Same as OIDC server + Fulcio.  OIDC cannot recover, Fulcio is not domain limited                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| x                      |                  |               |                       |              |               |              |             | Can become the Fulcio CA, Rekor, the Fulcio CA log, etc. Arbitrary software attack is trivial from there with MITM. Monitors may detect the attack, but Sigstore admins can’t recover without control of the root of trust.                                                                                                                                                                                                                                                                                                                              |
+
+
+## Mitigations and recommendations
+
+Here, we describe recommendations for using Sigstore in such a way that the likelihood and impact of a compromise are both mitigated.
+
+### 2FA on OIDC accounts
+As shown in the table above, compromise of an OIDC account is sufficient for an arbitrary software attack for any packages that account is trusted to sign. To mitigate this risk, we recommend reducing the likelihood of account compromise through the use of 2FA on all accounts that will be used with Sigstore. 2FA requires that not just a password, but a second factor like a YubiKey or authenticator app is also compromised. For maximum security, users should avoid less secure forms of 2FA such as text messages and phone calls. 
+
+OIDC does not provide a mechanism for checking if accounts have 2FA enabled, and so enforcement of this recommendation is up to individual signers in the ecosystem.
+
+### Identity monitors
+All certificates that are issued by Fulcio are written to a Certificate Transparency log, accessible at ctfe.sigstore.dev. For each artifact, Sigstore clients publish the artifact's hash, signature, and certificate. A signature verifier must check that an artifact has been uploaded to Rekor and the certificate has been published to Fulcio's log before trusting the artifact. This ensures that the artifact and certificate are publicly auditable, so that the identity owner can monitor the log to find unexpected occurrences of their identity.
```

Comment:
It's not clear from the text _where_ "Sigstore clients publish the artifact's hash, signature, and certificate".  Is this outside the Sigstore ecosystem, using their regular distribution channels?